crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -400,9 +400,10 @@ def services
 
       def openstackbackup
         crowbar_lib_dir = "/var/lib/crowbar"
-        dump_path = "#{crowbar_lib_dir}/upgrade/6-to-7-openstack_dump.sql"
+        dump_path = "#{crowbar_lib_dir}/backup/6-to-7-openstack_dump.sql.tar.gz"
         if File.exist?(dump_path)
           Rails.logger.debug("OpenStack backup already exists. Skipping...")
+          ::Crowbar::UpgradeStatus.new.end_step
```

Comment:
hmm i think we should rather keep it as 1 backup only, if we allow multiple backups we might run into some unforeseeable disk space issues, as the backups can be quite large (GB) in size